AutoDarkModeApp/Services/NavigationService.cs

```diff
@@ -115,12 +106,61 @@ private void OnNavigated(object sender, NavigationEventArgs e)
                 frame.BackStack.Clear();
             }
 
-            if (frame.GetPageViewModel() is INavigationAware navigationAware)
+            Navigated?.Invoke(sender, e);
+        }
+    }
+
+    private void OnBackRequested(NavigationView sender, NavigationViewBackRequestedEventArgs args) => GoBack();
+
+    private void OnItemInvoked(NavigationView sender, NavigationViewItemInvokedEventArgs args)
```

Comment:
For the future: Let's look at the approach on https://learn.microsoft.com/windows/apps/design/controls/navigationview#navigation